cmd/konvoy-image/cmd/create-package-bundle.go

```diff
@@ -0,0 +1,74 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	packageCmd        = "create-package-bundle"
+	packageCmdExample = "create-package-bundle --os redhat-8.4 --output-directory=artifacts"
+	validOS           = []string{
+		"centos-7.9",
+		"redhat-7.9",
+		"redhat-8.4",
+		"redhat-8.6",
+		"redhat-8.8",
+		"rocky-9.1",
+		"ubuntu-18.04",
+		"ubuntu-20.04",
+	}
+)
+
+type packageBundleCmdFlags struct {
+	targetOS          string
+	kubernetesVersion string
+	fips              bool
+	outputDirectory   string
+	containerImage    string
+}
+
+var packageBundleFlags packageBundleCmdFlags
+
+// NOTE Usage information always comes from the konvoy-image command, not the konvoy-image-wrapper command.
+// We define this command only to show usage; it does nothing else.
+var createPackageBundleCmd = &cobra.Command{
+	Use:     packageCmd,
+	Short:   "build os package bundles for airgapped installs",
+	Example: packageCmdExample,
+	Args:    cobra.NoArgs,
+	RunE: func(_ *cobra.Command, _ []string) error {
+		return nil
+	},
+}
+
+func init() {
+	fs := createPackageBundleCmd.Flags()
+	fs.StringVar(
+		&packageBundleFlags.targetOS,
+		"os",
+		"",
+		fmt.Sprintf("The target OS you wish to create a package bundle for. Must be one of %v", validOS),
+	)
+	fs.StringVar(
+		&packageBundleFlags.kubernetesVersion,
+		"kubernetes-version",
+		"",
```

Comment:
Are missing a default here?
I didn't see the version in the help text in `packageCmdExample`.

Reply:
Nope, we fetch this from the ansible variables when it’s not provided - so that is the default. 

It’s similar behavior to the build image workflow right now.

Reply:
Even better!